### ISSUE TITLE
Default version from v6 to v7

### ISSUE THREAD

**Issue by edigiacomo** [ARPA-SIMC/dballe#97]
Can V7 become the default version? See https://github.com/ARPA-SIMC/dballe/issues/94#issuecomment-318023226

**Comment by spanezz** [comment]
I would be very happy to make V7 the default version. Do we have the go ahead from @pat1 @dcesari and the others?

**Comment by pat1** [comment]
Nulla osta for V7 as default version.
If possible to use env var to force use of V6 version it will be a smooth transition option.
We have to solve the related bug connected to V7 version too.

Paolo, Davide, Andrea

**Comment by spanezz** [comment]
The only related bug I can see is #82, do you agree or did I miss something?

**Comment by edigiacomo** [comment]
Related to V7 I see: https://github.com/ARPA-SIMC/dballe/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20v7

**Comment by pat1** [comment]
la versione 6 pare ancora essere il default.

**Comment by edigiacomo** [comment]
@pat1 is right, V7 should be the default format starting from `v7.24`:

```
$ dnf info dballe | grep -i version
Version     : 7.24
$ dbadb wipe --dsn=sqlite://./test.sqlite3
$ sqlite3 test.sqlite3 "select * from dballe_settings;"
version|V6
```

It seems that V6 is still the default format in [7.24][1] and [7.25][2]

[1]: https://github.com/ARPA-SIMC/dballe/blob/v7.24-1/dballe/db/db.cc#L22
[2]: https://github.com/ARPA-SIMC/dballe/blob/v7.25-1/dballe/db/db.cc#L22

**Comment by edigiacomo** [comment]
Can we close this issue? Commit 3f4747c647d3997a6ff994b68511ec99fb949b3c was merged in `v7.26-1`.
